dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -167,10 +169,7 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
 
                             pushChannelReceived = co_await channelManager.CreatePushNotificationChannelForApplicationAsync();
 
-                            PushNotificationTelemetry::ChannelRequestedByApi(
-                                S_OK,
-                                AppModel::Identity::IsPackagedProcess(),
-                                remoteId);
+                            PushNotificationTelemetry::ChannelRequestedByApi(S_OK, remoteId);
```

Comment:
when going by the legacy approach - remoteId doesn't make sense. Can you distinguish between legacy and newer approach?

Reply:
yet, CreateChannelAsync was called with a remoteId (even if it'a a null one) and this is really all this is logging.

Reply:
What are the options we want to log beyond "legacy"?